resources/core/charts/cluster-users/templates/rbac-roles.yaml

```diff
@@ -75,7 +75,7 @@ metadata:
   annotations:
     "helm.sh/hook-weight": "0"
 rules:
-- apiGroups: ["", "api.kyma.cx", "apps", "authentication.kyma-project.io", "extensions", "gateway.kyma-project.io", "kubeless.io", "rbac.authorization.k8s.io", "applicationconnector.kyma-project.io", "servicecatalog.k8s.io", "servicecatalog.kyma.cx", "settings.k8s.io", "kyma.cx", "authentication.istio.io", "config.istio.io", "eventing.kyma.cx", "ui.kyma.cx", "ui.kyma-project.io", "batch", "extensions", "autoscaling"]
```

Comment:
I think kyma-view should have privileges to list metrics as well

Reply:
Added